settings.gradle.kts

```diff
@@ -81,6 +81,7 @@ dependencyResolutionManagement {
             val assertj = version("assertj", "3.24.2")
             val classgraph = version("classgraph", "4.8.165")
             val junit = version("junit", "5.10.1")
+            val junitSuite = version("junitSuite", "1.10.1")
```

Comment:
I think we don't need this line anymore?

Reply:
Good catch, leftover :)